.github/workflows/build.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-node@v3
         with:
-          node-version: 16
+          node-version: 18
```

Comment:
If you replace each of these lines with `node-version-file: '.nvmrc'` then it will stay in sync with that

Reply:
I have been slowly sprinkling it across our repos as I find them 😆